src/pages/transactions/[tid].tsx

```diff
@@ -0,0 +1,698 @@
+import Container from "@components/commons/Container";
+import GradientCardContainer from "@components/commons/GradientCardContainer";
+import { FiCopy, FiChevronUp, FiChevronDown } from "react-icons/fi";
+import NumericFormat from "@components/commons/NumericFormat";
+import { SearchBar } from "layouts/components/searchbar/SearchBar";
+import transactionDetailData from "mockdata/TransactionDetailData";
+import LinkText from "@components/commons/LinkText";
+import { truncateTextFromMiddle } from "shared/textHelper";
+import { Dispatch, SetStateAction, useState } from "react";
+import { useUnitSuffix } from "hooks/useUnitSuffix";
+import { InfoIcon } from "@components/icons/InfoIcon";
+import Tooltip from "@components/commons/Tooltip";
+import clsx from "clsx";
+import { ConfirmCheck } from "@components/icons/ConfirmCheck";
+import { RejectedCross } from "@components/icons/RejectedCross";
+import { GreenTickIcon } from "@components/icons/GreenTickIcon";
+
+const data = transactionDetailData.transactionDetailData;
+
+function Transaction() {
+  return (
+    <Container className="px-1 md:px-0 mt-12">
```

Comment:
we dont need this `Container` component anymore in main content as we wrapped parent component in  Container